cmd/do-agent/config.go

```diff
@@ -164,6 +164,20 @@ func initDecorator() decorate.Chain {
 	return chain
 }
 
+// initAggregatorSpecs initializes the field aggregation specifications.
+// The map's key is the prometheus metric name to aggregate over, and the value is the label to aggregate away.
+func initAggregatorSpecs() map[string]string {
+	aggregateSpecs := make(map[string]string)
```

Comment:
Can we move the if to the top to avoid allocations?
```
if config.dbaas == "" {
    return nil
}
```

Reply:
done!